db/warehouse_structure.sql

```diff
@@ -57760,7 +57761,7 @@ CREATE UNIQUE INDEX tx_id_ds_id_ft_idx ON public.financial_transactions USING bt
 -- Name: uidx_external_id_ns_value; Type: INDEX; Schema: public; Owner: -
 --
 
-CREATE UNIQUE INDEX uidx_external_id_ns_value ON public.external_ids USING btree (source_type, namespace, value) WHERE ((namespace)::text <> ALL ((ARRAY['ac_hmis_mci'::character varying, 'ac_hmis_mci_unique_id'::character varying])::text[]));
+CREATE UNIQUE INDEX uidx_external_id_ns_value ON public.external_ids USING btree (source_type, namespace, value) WHERE ((namespace)::text <> ALL (ARRAY[('ac_hmis_mci'::character varying)::text, ('ac_hmis_mci_unique_id'::character varying)::text]));
```

Comment:
@ttoomey or @gigxz do I need to re-run a migration?  This seems subtly different (but maybe just a difference in PG versions?)

Reply:
My guess is this is due to a difference in pg versions. It looks like the first line casts the whole array to text[] where as the second line casts each member to text individually. That seems equivalent, right?

Managing our schemas across multiple pg versions is a minor friction point

Reply:
Thanks.